mailbox/api/src/test/java/org/apache/james/mailbox/events/EventDeadLettersContract.java

```diff
@@ -0,0 +1,184 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.mailbox.events;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.apache.james.core.User;
+import org.apache.james.mailbox.Event;
+import org.apache.james.mailbox.MailboxListener;
+import org.apache.james.mailbox.MailboxSession;
+import org.apache.james.mailbox.model.MailboxConstants;
+import org.apache.james.mailbox.model.MailboxPath;
+import org.apache.james.mailbox.model.TestId;
+import org.junit.jupiter.api.Test;
+
+interface EventDeadLettersContract {
+
+    MailboxPath MAILBOX_PATH = new MailboxPath(MailboxConstants.USER_NAMESPACE, "user", "mailboxName");
+    User USER = User.fromUsername("user");
+    MailboxSession.SessionId SESSION_ID = MailboxSession.SessionId.of(235);
+    TestId MAILBOX_ID = TestId.of(563);
+    MailboxListener.MailboxAdded EVENT_1 = new MailboxListener.MailboxAdded(
+        SESSION_ID,
+        USER,
+        MAILBOX_PATH,
+        MAILBOX_ID,
+        Event.EventId.of("6e0dd59d-660e-4d9b-b22f-0354479f47b4"));
+    MailboxListener.MailboxAdded EVENT_2 = new MailboxListener.MailboxAdded(
+        SESSION_ID,
+        USER,
+        MAILBOX_PATH,
+        MAILBOX_ID,
+        Event.EventId.of("6e0dd59d-660e-4d9b-b22f-0354479f47b5"));
```

Comment:
Extract this eventId in a constant?

This helps readability as we will be able to easily see  that is is different.